src/platform/monitoring/external-services/actions.js

```diff
@@ -7,9 +10,21 @@ export const LOADING_BACKEND_STATUSES = 'LOADING_BACKEND_STATUSES';
 const BASE_URL = '/backend_statuses';
 
 export function getBackendStatuses() {
-  return dispatch => {
+  return (dispatch, state) => {
     dispatch({ type: LOADING_BACKEND_STATUSES });
 
+    // create global downtime data if feature toggle is enabled and if
+    // current time is in downtime window
+    // default to empty array
+    const { downtimeStart, downtimeEnd } = scheduledDowntimeWindow;
+    const globalDowntimeActive =
+      toggleValues(state).vaGlobalDowntimeNotification &&
+      moment().isAfter(downtimeStart) &&
+      moment().isBefore(downtimeEnd);
+    dispatch({
```

Comment:
ExternalServicesError follows a slightly different pattern. it doesn't use windows- it deals with strictly a down or up state.

Reply:
It looks like we could share some logic here with downtime notifications, but I don't know if we're trying to keep the implementations completely separate anyway.

Reply:
I added a ticket for reconciling this. The concerns overlap heavily. seems like the main difference is that one provides an arbitrary timeframe and one provides the current state (no start or end time). I don't think we need to have all of this plumbing for that sort of temporal difference.